astroquery/simbad/core.py

```diff
@@ -1072,12 +1141,13 @@ def query_bibcode(self, bibcode, *, wildcard=False,
 
         query += " ORDER BY bibcode"
 
-        return self.query_tap(query, get_query_payload=get_query_payload)
+        return self.query_tap(query, get_query_payload=get_query_payload,
+                              async_job=async_job)
 
     @deprecated_renamed_argument(["verbose", "cache"], new_name=[None, None],
                                  since=['0.4.8', '0.4.8'], relax=True)
-    def query_objectids(self, object_name, *, verbose=None, cache=None,
-                        get_query_payload=False, criteria=None):
+    def query_objectids(self, object_name, *, criteria=None, get_query_payload=False,
+                        async_job=False, verbose=None, cache=None):
```

Comment:
why did you reorder the kwargs here?   I think since they're forced to be kwargs, the order doesn't matter, just curious.

Reply:
Because the last two are deprecated (verbose, and cache) so I thought it would be nicer for the users reading through the method's signature